src/app/components/shared/datetime-formats/time-since/time-since.component.ts

```diff
@@ -36,7 +36,7 @@ export class TimeSinceComponent
       // we want to update the "time since" every second if the time is less than a minute
```

Comment:
i thought of a better algorithm for this:

- on subscribe, store the lastChange date in a private field in the component
- on each tick
    - let now be Date.now()
    - let valueDelta be the difference between now and value
    - let lastChangeDelta be the difference between now and lastChange
    - compare the absolute delta and decide if we're lessThanAMinute or not
    - if we're lessThanAMinute
        - then update()
        - and set lastChange to now
    - else if lastChangeDelta >= 60
        - then update()
        - and set lastChange to now
    - else 
        -  // do nothing

Reply:
Slightly simplified implementation that consolidates `update()` calls

![image](https://github.com/QutEcoacoustics/workbench-client/assets/33742269/f6f734d8-64a5-4490-a5a7-79b8df536700)

Reply:
it's clean, but the reason we pinned date.now is because we will see very minor drift between when we calculate lastChange and when we set it.